<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>机体参数</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Sup_Hyp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>331</width>
     <height>166</height>
    </rect>
   </property>
   <property name="title">
    <string>超音速/高超声速参数</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_BNOSE">
        <property name="text">
         <string>BNOSE：机头样式</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_BNOSE">
        <item>
         <property name="text">
          <string>1.0 圆锥型机头</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0 尖型机头</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_BTAIL">
        <property name="text">
         <string>BTAIL：机尾样式</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_BTAIL">
        <item>
         <property name="text">
          <string>1.0 圆锥型机尾</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2.0 尖型机尾</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_BLN">
        <property name="text">
         <string>BLN：机身机头长度</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BLN">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_BLA">
        <property name="text">
         <string>BLA：圆柱状后机身部分长度</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BLA"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QCheckBox" name="checkBox_DS">
        <property name="text">
         <string>DS：机头钝度</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_DS"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_Sub">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>338</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>亚音速参数</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>机身横截面参数组合</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_VarCombo">
        <item>
         <property name="text">
          <string>R-ZU-ZL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S-ZU-ZL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R-S-P-ZU-ZL  </string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QCheckBox" name="checkBox_ELLIP">
        <property name="text">
         <string>ELLIP：机身高宽比</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>88</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="ELLIP"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>METHED：计算方法</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_METHED">
        <item>
         <property name="text">
          <string>1. 使用现有方法（系统默认）</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2. 使用Jorgensen方法</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_3</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QTableWidget" name="Tab_ComboVariables">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>331</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>664</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="image_BODY">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="card_rc.qrc">:/card/rc_card/Body_SubSonic.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string>本节主要录入机体的参数
NX : 机身定义位置的数量，最大20
X  : 机身截面的纵向距离，最大20
S  : 横截面面积（cross sectional area）
P  : 周长（periphery）
R  : 半径（local planform half width）
ELLIP : ELLIP = 高度A/宽度B,&gt;1表示高而窄
ZU : 截面处机体上表面的Z坐标（中线以上为正）
ZL : 截面处机体下表面的Z坐标（中线以下为负）

Tips 1:截面参数模式
A1. 只使用R或者S，其他参数按照圆截面估计；
A2. 如果ELLIP被定义，从R或者S估计其他参数，ELLIP = 高度A/宽度B,&gt;1表示高而窄；此处必须使用METHOD=2.0
A3. R,S,P都定义
Tips 2;参考截面说明
参考平面 = 轴对称机体的堆成轴
Tips 3：
机体中线线处Z=0
Tips 4：
BNOSE BTAIL BLN BLA 这些参数在亚音速域不需要
Tips 5：
DS 只在高超声速需要 （Hypersonic Speed Regime Oonly）

Tips 6：
在跨声速和超声速是由S得到等效半径 Req = sqrt(S/π)

Tips 7：在Lbt =0 是忽略BTAIL，即无尾时</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>布局选择</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>180</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_SpeedRegime">
      <item>
       <property name="text">
        <string>亚音速</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>跨音速</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>超音速</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>高超音速</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>NX：机身横截面的个数</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="NX"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>303</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QCheckBox" name="checkBox_ITYPE">
      <property name="text">
       <string>ITYPE:跨音速阻力分量马赫数计算方法</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>68</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>305</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>68</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_ITYPE">
      <item>
       <property name="text">
        <string>1. 直翼，不计算面积效应(Area Rule)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2. 掠翼，不计算面积效应(系统默认)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3. 掠翼，计算面积效应</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_SpeedRegime</tabstop>
  <tabstop>NX</tabstop>
  <tabstop>comboBox_VarCombo</tabstop>
  <tabstop>checkBox_ELLIP</tabstop>
  <tabstop>ELLIP</tabstop>
  <tabstop>comboBox_BNOSE</tabstop>
  <tabstop>comboBox_BTAIL</tabstop>
  <tabstop>lineEdit_BLN</tabstop>
  <tabstop>lineEdit_BLA</tabstop>
  <tabstop>checkBox_DS</tabstop>
  <tabstop>lineEdit_DS</tabstop>
  <tabstop>comboBox_METHED</tabstop>
  <tabstop>checkBox_ITYPE</tabstop>
  <tabstop>comboBox_ITYPE</tabstop>
  <tabstop>Tab_ComboVariables</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="card_rc.qrc"/>
  <include location="card_rc.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>slot_Tab_ComboVariables()</slot>
  <slot>slot_super_hyper()</slot>
 </slots>
</ui>
